src/corelocation.cs

```diff
@@ -431,6 +441,15 @@ partial interface CLLocationManager {
 		[Export ("authorizedForWidgetUpdates")]
 		bool IsAuthorizedForWidgetUpdates { [Bind ("isAuthorizedForWidgetUpdates")] get; }
 
+		[Async]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15, 0)]
```

Comment:
Missing Catalyst. It's not excluded on type-level so it must be annotated for new API.